modules/brightcomBidAdaper.js

```diff
@@ -0,0 +1,242 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+import { config } from '../src/config';
+
+const BIDDER_CODE = 'brightcom';
+const URL = 'https://brightcombid.marphezis.com/hb';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid,
+  buildRequests,
+  interpretResponse,
+  getUserSyncs
+};
+
+function buildRequests(bidReqs, bidderRequest) {
+  try {
+    const loc = utils.getTopWindowLocation();
```

Comment:
This function is eventually going to be deprecated, so it shouldn't be used if possible.  

We have setup an alternative feature with this type of information that can be used instead.  The associated domain data can be accessed via the `bidderRequest.refererInfo`object.  

The following page provides some additional details on the object and its data points if you want to review further:  
http://prebid.org/dev-docs/bidder-adaptor.html#referrers